frontend/app/(tabs)/create-recipe.tsx

```diff
@@ -1,92 +1,81 @@
-import { useRouter, Link, useLocalSearchParams } from "expo-router";
-import React, { useEffect } from "react";
+import React from "react";
 import {
   useForm,
   useFieldArray,
   Controller,
-  UseFieldArrayReturn,
 } from "react-hook-form";
 import { Text, View, StyleSheet, TextInput, Button, Alert } from "react-native";
 import { ScrollView } from "tamagui";
-import { useQueryClient, useQuery } from "@tanstack/react-query";
 import axios from "axios";
-
-interface FormValues {
-  name: string;
-  description: string;
-  default_servings: string;
-  mins_prep: string;
-  mins_cook: string;
-  procedure: string;
-  tags: { category: string; value: string }[];
-  ingredients: { quantity: string; units: string; value: string }[];
-}
+import {Recipe, FormattedRecipe, emptyData, sampleData} from "../../components/create-recipe-types/create-recipe-helper"
```

Comment:
Thank you for adding a sample data that I can fill in when editing recipes, this makes my work a lot less of a headache.

Reply:
No problem!